ff/serving/api/api.go

```diff
@@ -109,23 +149,29 @@ func (serv *ApiServer) Serve() error {
 	opts := []grpc.DialOption{
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
 	}
-	conn, err := grpc.Dial(serv.metaAddr, opts...)
+	metaConn, err := grpc.Dial(serv.metadata.address, opts...)
```

Comment:
could these be done in paralell?

Reply:
Does it need to be? Its just opening a channel right?

Reply:
it doesn't just saves time i imagine, nbd tho

Reply:
Gonna leave it for now since it only runs at startup and dont want to mess with it too much atm